PAbishai/Ride-My-Way
# #158423266 API to create a join request

**What does this PR do**
Make API to fetch all ride offers

**Description of tasks to be completed?**
POST /api/v1/rides/<ride_id>/requests should add a ride request

**How should this be manually tested?**
Clone the repo
Create a virtualenv - **_#virtualenv Ride-My-Way_**
cd into the repo and activate it
run **_pip install -r requirements.txt_**
cd into the test folder
run **_pytests_**
the \test should pass
comment out some functions in api_rides.py
run pytest again
the test should fail
cd out of the \test folder and into the \resources folder
run **_python api_rides.py_**
use postman and POST to localhost:5000/api/v1/rides/<ride_id>/requests or the port you have for flask
data should be added in raw json format
It should give a status code of 201 and a json response with the request

**What are the relevant pivotal tracker stories**
[#158423266](https://www.pivotaltracker.com/story/show/158423266)

**Screenshots**
How data should be added
![screenshot from 2018-06-26 06-41-25](https://user-images.githubusercontent.com/12625365/41888906-5d803bd2-7910-11e8-9981-3a3fb0bffd6d.png)
